cfgov/unprocessed/js/organisms/Header.js

```diff
@@ -49,26 +49,26 @@ function Header( element ) {
     // Semi-opaque overlay that shows over the content when the menu flies out.
     _overlay = overlay;
 
+    // Don't initialize the mega menu if it isn't on the page.
+    if ( _dom.classList.contains( `${ BASE_CLASS }__mega-menu` ) ) {
```

Comment:
How about this instead:

```
// Don't initialize the mega menu if it isn't on the page.
    _globalSearch = new GlobalSearch( _dom );

    if ( _dom.classList.contains( `${ BASE_CLASS }__mega-menu` ) ) {
      _megaMenu = new MegaMenu( _dom );
      _megaMenu.addEventListener( 'rootExpandBegin', _megaMenuExpandBegin );
      _megaMenu.addEventListener( 'rootCollapseEnd', _megaMenuCollapseEnd );
      _megaMenu.init();

      // If we have a mega menu, it needs to be collapsed when search is expanded.
      _globalSearch.addEventListener( 'expandBegin', _megaMenu.collapse );
    }

    _globalSearch.init();
```

Reply:
Thanks, that is nicer. Applied in 844a499.